groupy/resources.py

```diff
@@ -71,7 +78,17 @@ def from_payload(cls, payload):
             payload["data"]["user"]["public_keys"],
             payload["data"]["permissions"],
             payload["data"]["user"]["metadata"],
-            payload["data"]["user"]["enabled"]
+            payload["data"]["user"]["enabled"],
+
+            # New values may not exist in the JSON objects, so we need to be careful.
+            payload["data"]["user"].get("passwords", []),
+
+            # Optional field only present for service accounts.
+            #
+            # TODO(rra): ServiceAccount objects should lift these up to top-level properties and
+            # User objects should not have this, and we should return a ServiceAccount when
+            # retrieving a User with this set.
+            payload["data"]["user"].get("service_account"),
```

Comment:
it shouldn't matter as long as it's used as a boolean, but this will return None if the key isn't there.

Reply:
Yeah, that's intentional -- I thought that would be most sensible value for the `service_account` property to take, at least until I rework all of this so that this uses polymorphism.  Do you think an empty hash would be better?

Reply:
Empty dict, rather. (Hash is the Perl terminology.)

Reply:
my bad, for some reason I thought that was supposed to be a boolean. I agree that `None` makes more sense for a dictionary.